### DISCUSSION TITLE
Stream-chat-flutter UI kits on web and desktop

### DISCUSSION THREAD

**Discussion by theweiweiway** [GetStream/stream-chat-flutter#500] [General]
Hi!  

It's there a plan to make the UI kits on the stream-chat-flutter library available to web and desktop as well?

Thanks!

**Comment by imtoori** [comment]
Turning this into a discussion if you don't mind

**Reply by imtoori** [reply]
Yes! It's in our priorities for the next quarter :) 

You can already use the stream_chat and stream_chat_flutter_core packages on both web and desktop, but the stream_chat_flutter one is not optimized for that yet

**Reply by theweiweiway** [reply]
Sweet, good to know. Thanks!
